### ISSUE TITLE
Warning related to libstdc++

### ISSUE THREAD

**Issue by openjck** [hayd/deno-docker#113]
I'm seeing the following on a Linux host:

```
$ sudo docker run -it --init hayd/alpine-deno:1.8.0 eval 'console.log("Hello, world!");'
deno: /usr/lib/libstdc++.so.6: no version information available (required by deno)
deno: /usr/lib/libstdc++.so.6: no version information available (required by deno)
deno: /usr/lib/libstdc++.so.6: no version information available (required by deno)
deno: /usr/lib/libstdc++.so.6: no version information available (required by deno)
deno: /usr/lib/libstdc++.so.6: no version information available (required by deno)
Hello, world!
```

I know this is related to https://github.com/denoland/deno/issues/9686 and that libstdc++ is installed manually to work around it, however, it seems the workaround is not working in my case for some reason.

**Comment by hayd** [comment]
Sadly I am not sure how to hide those warning lines. Does deno still run normally aside from those 5 (!) lines?

**Comment by openjck** [comment]
So far, yes! I haven't used it extensively yet, but it seems to work fine.

**Comment by HanKruiger** [comment]
This issue (in a different context) seems identical or at least related : https://github.com/sgerrand/alpine-pkg-glibc/issues/80

**Comment by jsejcksn** [comment]
Is this only an issue on the `alpine` build or does it affect other images as well?

**Comment by hayd** [comment]
Pretty sure this is only alpine here, but any distribution which has an unnamed build of libstdc++ will see the same issue.

**Comment by martijnvanschie** [comment]
Hi, I'm getting the same error on my docker image using `hayd/alpine-deno:latest`. After that I also get the following error:

```javascript
paint-manager_api  | error: Uncaught (in promise) TypeError: Cannot read property 'router' of undefined
paint-manager_api  |       if (!middleware.router) {
paint-manager_api  |                       ^
paint-manager_api  |     at Router.#register (https://deno.land/x/oak@v7.4.1/router.ts:404:23)
paint-manager_api  |     at Router.#useVerb (https://deno.land/x/oak@v7.4.1/router.ts:492:19)
paint-manager_api  |     at Router.options (https://deno.land/x/oak@v7.4.1/router.ts:741:18)
paint-manager_api  |     at file:///app/brands/brands.routes.ts:7:4
```

Could this be related. The Deno app runs fine from the commandline.

Thanks.

**Comment by danopia** [comment]
Deno 1.10 is out (as 1.10.1) and libstdc++ is no longer required when building an Alpine image.

Once this repo is updated to 1.10 the warning should be gone as well. 🎉

---

> Could this be related. The Deno app runs fine from the commandline.

@martijnvanschie This looks to me like an unrelated issue. It looks like your application gave an `undefined` to oak.

**Comment by martijnvanschie** [comment]
@danopia I checked the oak Github following the stack trace and its on the `RouterMiddleware` instance in the router class.

I reported an issue in the oak repo.

Its just that a local `docker run` version starts normally so just wondering if the image might somehow be missing something Deno related.

**Comment by hayd** [comment]
yes, fixed in 1.10.1.  and dep removed by @kinghat 💪

**Comment by martijnvanschie** [comment]
@hayd yes, just tested new build and no more warning 👍
